transmission_rpc/torrent.py

```diff
@@ -542,7 +542,7 @@ def peers_connected(self) -> int:
         return self.fields["peersConnected"]
 
     @property
-    def peers_from(self) -> int:
+    def peers_from(self) -> dict:
```

Comment:
lucky v8 is not released yet, please add a typing class to annotate all properties

Reply:
Done